proto/src/main.rs

```diff
@@ -0,0 +1,54 @@
+extern crate clap;
+extern crate protobuf;
+use clap::{App, Arg};
+use protobuf::*;
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::{BufReader, Write};
+
+mod person;
+use person::Person as person_obj;
```

Comment:
rust is not object oriented, using terminologies like person obj...doesn't make much sense

Reply:
i'll prefer Person as it is